Libraries/Components/SliderIOS/SliderIOS.ios.js

```diff
@@ -76,6 +76,11 @@ var SliderIOS = React.createClass({
      */
     disabled: PropTypes.bool,
 
+   /**
+     * Sets an image for the track. It only supports images that are included as assets
+     */
+    trackImage: PropTypes.string,
```

Comment:
This should be an ImageSource instead of a string. That makes it possible to use the `require()` syntax for setting the image, and also lets you specify a `scale` property if you use a base64 url.